Dockerfile

```diff
@@ -1,8 +1,30 @@
-FROM tomcat:9-jre8
+FROM tomcat:9-jdk11
 MAINTAINER James Hiebert <hiebert@uvic.ca>
 
-ENV NCWMS_VERSION 2.2.5
+# Add Tomcat User/Group
+ARG RUN_USER=tomcat
+ARG RUN_GROUP=tomcat
+RUN groupadd -r ${RUN_GROUP} && \
+    useradd -g ${RUN_GROUP} -d ${CATALINA_HOME} -s /bin/bash ${RUN_USER} && \
+    chown -R tomcat:tomcat $CATALINA_HOME && \
+    mkdir -p /usr/local/tomcat/.ncWMS2 && \
+    chown tomcat:tomcat /usr/local/tomcat/.ncWMS2
+USER ${RUN_USER}
 
-RUN curl -L https://github.com/Reading-eScience-Centre/ncwms/releases/download/ncwms-$NCWMS_VERSION/ncWMS2.war > $CATALINA_HOME/webapps/ncWMS.war
+# Download and unpack the full EPSG geodetic dataset
+ADD --chown=tomcat:tomcat \
+    https://repo1.maven.org/maven2/org/apache/sis/non-free/sis-embedded-data/1.0/sis-embedded-data-1.0.jar \
+    ./sis-embedded-data-1.0.jar
+RUN unzip sis-embedded-data-1.0.jar && \
+    rm sis-embedded-data-1.0.jar
+# Unpacking the jar creates a *directory* called SIS_DATA containing the dataset.
+# The *env variable* SIS_DATA tells ncWMS where to find it.
+ENV SIS_DATA=/usr/local/tomcat/SIS_DATA
+
+# Download the ncWMS 2.5.1 snapshot (pre-release version).
+# Note error in version number in release name: 1.5.1 should be 2.5.1.
```

Comment:
We know what you mean, but it may be more clear specify that 2.5.1 was erroneously released with as version 1.5.1. I.e. the mistake is in the release, not in this this Dockerfile.